src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -121,6 +121,24 @@ public boolean clickEditInstructorLink(int instrNum) {
         return isEditable;
     }
     
+    public void clickCancelEditInstructorLink(int instrNum) {
+        getCancelEditInstructorLink(instrNum).click();
+    }
+    
+    public boolean isInstructorFormNotEditable(int instrNum) {
+        WebElement saveButton = getSaveInstructorButton(instrNum);
+        waitForElementToDisappear(By.id("btnSaveInstructor" + instrNum));
+        
+        WebElement editInstructorNameTextBox = getNameField(instrNum);
+        WebElement editInstructorEmailTextBox = getEmailField(instrNum);
+        
+        boolean isNotEditable = !editInstructorNameTextBox.isEnabled()
+                             && !editInstructorEmailTextBox.isEnabled()
+                             && !saveButton.isDisplayed();
```

Comment:
Shifting the `&&`s 3 spaces to the right to be in line with `!e` in the line above would be good :)